### ISSUE TITLE
package builds are broken in 18.6.0

### ISSUE THREAD

**Issue by Conan-Kudo** [ascherer/debbuild#85]
With debbuild 18.6.0, I get these errors:

```
+ /bin/cp -pr dist/copyright /home/ngompa/debbuild/BUILDROOT/foo-1-1debian9.amd64/usr/share/doc/foo/
sh: 1: %{__mkdir}: not found
/bin/cp: cannot create regular file '/home/ngompa/debbuild/BUILDROOT/foo-1-1debian9.amd64/main/usr/share/doc/foo/': No such file or directory
sh: 1: %{__mkdir}: not found
/bin/cp: cannot create directory '/home/ngompa/debbuild/BUILDROOT/foo-1-1debian9.amd64/main/usr/lib/': No such file or directory
sh: 1: %{__mkdir}: not found
/bin/cp: cannot create symbolic link '/home/ngompa/debbuild/BUILDROOT/foo-1-1debian9.amd64/main/usr/bin/': No such file or directory
sh: 1: %{__mkdir}: not found
/bin/cp: cannot create regular file '/home/ngompa/debbuild/BUILDROOT/foo-1-1debian9.amd64/main/etc/cron.d/': No such file or directory
sh: 1: %{__mkdir}: not found
sh: 1: cannot create /home/ngompa/debbuild/BUILDROOT/foo-1-1debian9.amd64/main/usr/share/doc/foo/changelog.gz: Directory nonexistent
```

**Comment by Conan-Kudo** [comment]
This was caused by the change to `%__mkdir_p` in https://github.com/ascherer/debbuild/commit/33d846ad8067f14eda9ce4abe3f85aef6cab474a.

Reverting that change fixes building packages.

**Comment by ascherer** [comment]
Curiously, `rpmbuild` fails similarly if `%__mkdir_p` is redefined in its `/usr/lib/rpm/macros`. I wonder how `%__id_u` is different?

**Comment by ascherer** [comment]
Fixed in 18.6.1.
